src/coprocessor/executor/mod.rs

```diff
@@ -27,18 +36,28 @@ pub mod aggregation;
 
 #[allow(dead_code)]
 pub struct ExprColumnRefVisitor {
-    pub col_ids: HashSet<i64>,
+    cols_offset: HashSet<usize>,
+    cols_len: usize,
 }
 
 #[allow(dead_code)]
 impl ExprColumnRefVisitor {
-    pub fn new() -> ExprColumnRefVisitor {
-        ExprColumnRefVisitor { col_ids: HashSet::new() }
+    pub fn new(cols_len: usize) -> ExprColumnRefVisitor {
+        ExprColumnRefVisitor {
+            cols_offset: HashSet::default(),
```

Comment:
No associated item named `with_capacity` found for type `std::collections::HashSet<_, std::hash::BuildHasherDefault<fnv::FnvHasher>>` in the current scope @BusyJay